parsec/core/mountpoint/winfsp_operations.py

```diff
@@ -233,12 +233,6 @@ def close(self, file_context):
         if isinstance(file_context, OpenedFile):
             self.fs_access.fd_close(file_context.fd)
```

Comment:
Is it ok to do `fd_close` on a deleted file ? (should it trigger an exception ?)

Anyway I think a comment about this would be good here ^^

Reply:
It raises a `FSInvalidFileDescriptor` (`errno.EBADF`, `ntstatus.STATUS_INVALID_HANDLE`)

I don't know if it's a problem or not, the test suite seems to agreee with it though :thinking:

Reply:
Actually deleting a file does not delete the underlying manifest, it simply removed this entry from the parent manifest.

Reply:
b6876b9